### ISSUE TITLE
New release please?

### ISSUE THREAD

**Issue by rlam3** [stewartpark/Flask-JSGlue#8]
New release please?

**Comment by stewartpark** [comment]
Hi rlam3,

I've been getting some suggestions and applying it to the source code.
I'm planning a release this month, but if you need certain fixes after the latest release, I can make an interim version on pip, or you can pip install the git repository like below:

```
$ pip install git+https://github.com/stewartpark/Flask-JSGlue
```

Thanks,
Stewart

**Comment by JoshRosen** [comment]
+1 on making a new release to incorporate #9, since that's a pretty important fix.

**Comment by rlam3** [comment]
@JoshRosen  Its been awhile since this has been updated/supported. It doesn't look like this project is supported anymore. I'd recommend that you take the lead in creating a new pip instead :)

**Comment by stewartpark** [comment]
Hi guys,

I just released a [new version (0.3.1)](https://pypi.python.org/pypi/Flask-JSGlue/0.3.1) with the changes that didn't get to release for a long time. I'm sorry that I've been procrastinating. X_X
